bonfire/bonfire.py

```diff
@@ -1137,23 +1159,21 @@ def _err_handler(err):
 
 
 @reservation.command("extend")
-@click.option(
-    '--namespace',
-    '-ns',
-    type=str,
-    default=None,
-    help='Name of the reserved namespace',
-)
+@options(_reservation_lookup_options)
```

Comment:
Seeing double options on this command now:
```
❯ bonfire reservation extend --help
Usage: bonfire reservation extend [OPTIONS]

Options:
  --name TEXT           Identifier for the reservation
  -r, --requester TEXT  Name of the user requesting a reservation
  -n, --namespace TEXT  Namespace for the reservation
  --name TEXT           Identifier for the reservation
  -r, --requester TEXT  Name of the user requesting a reservation
  -d, --duration TEXT   Duration of the reservation
  -h, --help            Show this message and exit.
```

Reply:
Is it because of `@options(_reservation_process_options)` below?

Reply:
Yes. I thought they might collapse together, but I've updated it to only use one group and pass in an extra option separately